src/AspNet.Security.OAuth.Odnoklassniki/OdnoklassnikiAuthenticationExtensions.cs

```diff
@@ -0,0 +1,76 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System;
+using AspNet.Security.OAuth.Odnoklassniki;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Authentication;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods to add Odnoklassniki authentication capabilities to an HTTP application pipeline.
+    /// </summary>
+    public static class OdnoklassnikiAuthenticationExtensions
+    {
+        /// <summary>
+        /// Adds <see cref="OdnoklassnikiAuthenticationHandler"/> to the specified
+        /// <see cref="AuthenticationBuilder"/>, which enables Odnoklassniki authentication capabilities.
+        /// </summary>
+        /// <param name="builder">The authentication builder.</param>
+        /// <returns>The <see cref="AuthenticationBuilder"/>.</returns>
+        public static AuthenticationBuilder AddOdnoklassniki([NotNull] this AuthenticationBuilder builder)
+        {
+            return builder.AddOdnoklassniki(OdnoklassnikiAuthenticationDefaults.AuthenticationScheme, options => { });
+        }
+
+        /// <summary>
+        /// Adds <see cref="OdnoklassnikiAuthenticationHandler"/> to the specified
+        /// <see cref="AuthenticationBuilder"/>, which enables Odnoklassniki authentication capabilities.
+        /// </summary>
+        /// <param name="builder">The authentication builder.</param>
+        /// <param name="configuration">The delegate used to configure the OpenID 2.0 options.</param>
```

Comment:
I imagine this is copied from elsewhere that also needs fixing, but I've just noticed this should be _"OAuth 2.0"_, not _"OpenID 2.0"_.